app/views/chugs/_podium_list.html.erb

```diff
@@ -0,0 +1,91 @@
+<ul role="list" class="divide-y divide-gray-200">
+  <% if chugs.count >= 3 %>
+    <% chugs.each_with_index do |chug, index|  %>
+      <% if index == 0 %>
+        <li class="flex items-center justify-center">
+          <div class="podium py-4">
+            <img class="medal gold" src="/img/medal_gold.png" alt="Gold medal">
```

Comment:
Moeten deze in /public zitten, omdat ik nu pas zie dat de meeste in /app/assets zitten

Reply:
Hier gaat het dan dus over de .png bestanden in de public folder

Reply:
Nou ik heb ze daar dus ook maar ingedaan nu

Reply:
Ze horen in de assets folder maar dan kun je de url niet op deze manier gebruiken, dat gaat volgens mij lokaal nog wel goed maar op productie niet. 

Als de file in `app/assets/images/` zit: 
```
<%= image_tag "medal_gold.png", class: "medal gold", alt: "Gold medal" %> 
```

Reply:
Check, dat is ook gefixt in de 'replace medal location and use image_tag erb code' commit

Reply:
Ik zag dat je het zelf al gevonden had idd :)